apps/sms/views/conversation/test/unit/recipients_test.js

```diff
@@ -143,6 +143,24 @@ suite('Recipients', function() {
       assertDeepEqual(recipient, fixture);
     });
 
+    test('recipients.add() parenthesis ', function(done) {
+        recipients.on('add', function(count, added) {
+          assert.isFalse(added.isQuestionable);
+          recipients.off('add');
+          done();
+        });
+
+        recipients.add({
```

Comment:
Also it would be great if you can add separate ```test``` for the leading ```+``` as well if it doesn't exist in this file.

Thanks!